app/user/rest.py

```diff
@@ -593,7 +593,7 @@ def update_password(user_id):
     try:
         _update_alert(user, change_type)
     except Exception as e:
-        current_app.logger.error(e)
+        current_app.logger.exception(e)
```

Comment:
I'm being lazy and not wanting to look it up for myself - will `logger.exception` automatically supply the exception as an argument?  Sometimes I see you're passing in the exception, other times I see you just pass in a string

Reply:
I had the same question and experimented in the interpreter.  Calling "exception" without any arguments is an error.  Passing the exception is equivalent to passing `str(e)`.